README.md

```diff
@@ -1,27 +1,30 @@
-# SMASH - Smart Mobile App for Surveyor's Happiness
+# SMASH — Smart Mobile App for Surveyor Happiness
```

Comment:
Hi, I actually want it to be the happiness of the surveyor, hence surveyor's.

Why do you feel it is wrong?

Reply:
Need "for the surveyor's" with that. or
"smart Mobile App for the Happiness of surveyors"

Reply:
Ok, then I would go with "the surveyor's" to stick with the Acronym.